### DISCUSSION TITLE
What Syntax Highlighter does Bootstrap use?

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#41041] [General] [answered]
Hi. I know this isn't exactly a question/issue about Bootstrap, but I just really want to know. To be specific, the Syntax Highlighter used on the Bootstrap website.

[Here's an example](https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start) of what I'm talking about, where the code on the page is highlighted really nicely.

What Syntax highlighter is used, and what theme for the Syntax Highlighter is used (if there is one).

**Comment by julien-deramond** [comment, marked as answer]
Hi @Skrillx13 
Our documentation is built with [Hugo](https://gohugo.io) which uses [Chroma](https://github.com/alecthomas/chroma) as syntax highlighter (source: https://gohugo.io/content-management/syntax-highlighting/)
Then, we override the style with this [site/assets/scss/_syntax.scss](https://github.com/twbs/bootstrap/blob/main/site/assets/scss/_syntax.scss) file for light and dark mode, based on Bootstrap color palette.

**Reply by ghost** [reply]
Thanks a lot! So if I were to want to use Bootstraps syntax highlighting colours, I would need to override the Chroma styles with the _syntax.scss file right

**Reply by julien-deramond** [reply]
Yeah, 1st step would be to find a way to use Chroma in your project, and then follow its documentation to see how to override properly the rendering (the colors), maybe there's a way to configure it with a config file, IDK. Worst case scenario, it's going to be something like what we've done with the `_syntax.scss` "from outside".

**Comment by jonhubby** [comment]
The Bootstrap website uses Prism.js for syntax highlighting, which is pretty popular. As for the theme, they typically use a custom version of the "Solarized Light" theme. It gives that clean look to the code snippets.
